CHANGELOG.md

```diff
@@ -6,6 +6,38 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 This project mirrors major Elm versions. So version 0.18.\* of this project will
 be compatible with Elm 0.18.\*.
 
+## 0.19.1-beta1.revision5 - 2020-TBA
```

Comment:
Note that according to the `semver` module:

```
0.19.1-beta1.revision5 < 0.19.1-revision5

0.19.1-revision5.beta1 > 0.19.1-revision5
```

What I’m trying to say is that we need to be careful when choosing the suffix so tools like `ncu`/`npm-upgrade` don’t think that the beta is a later version than the final `revision5`.

Reply:
Lets not do a beta release. I think the quality is pretty good, I have been using the master with no issues.